### PR TITLE
feat: add project_urls example

### PR DESCRIPTION
**Pull request by henryiii**
In https://github.com/pypa/packaging.python.org/pull/823, it was mentioned that `project_urls` might be a good thing to show. This adds that example.

Happy to change this as needed, or if it's seen as "too much", that's fine too, it's just a followup to https://github.com/pypa/packaging.python.org/pull/823#discussion_r571482624 as promised in https://github.com/pypa/packaging.python.org/pull/823#discussion_r571484257 . I don't think I've used it in setup.py, I think I got the setup.py version right...

### CODE REVIEW

File: source/tutorials/packaging-projects.rst [hunk 1]
```diff
@@ -234,6 +238,9 @@ an escape hatch when absolutely necessary.
             long_description=long_description,
             long_description_content_type="text/markdown",
             url="https://github.com/pypa/sampleproject",
+            project_urls={
```
Comment (souldia18) [inline source/tutorials/packaging-projects.rst:241]: 224

**Review by souldia18 — APPROVED** [review]

File: source/tutorials/packaging-projects.rst (hunk 1 above)
Comment (henryiii) [reply source/tutorials/packaging-projects.rst:241]: ?

File: source/tutorials/packaging-projects.rst [hunk 2]
```diff
@@ -169,6 +171,8 @@ an escape hatch when absolutely necessary.
     - ``url`` is the URL for the homepage of the project. For many projects, this
       will just be a link to GitHub, GitLab, Bitbucket, or similar code hosting
       service.
+    - ``project_urls`` lets you list any number of extra links to show in PyPI.
```
Comment (webknjaz) [inline source/tutorials/packaging-projects.rst:174]:
```suggestion
    - ``project_urls`` lets you list any number of extra links to show on PyPI.
```

File: source/tutorials/packaging-projects.rst [hunk 3]
```diff
@@ -266,6 +273,8 @@ an escape hatch when absolutely necessary.
...
```
Comment (webknjaz) [inline source/tutorials/packaging-projects.rst:276]:
```suggestion
    - ``project_urls`` lets you list any number of extra links to show on PyPI.
```

**Review by webknjaz — APPROVED** [review]
Looks lovely! There's just one small typo that would be great to address.

**Pull request merged**